homework.md

```diff
@@ -92,19 +92,19 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save your changes
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+CYF Blocks
```

Comment:
This is not the correct place that we edit the repos that we fork

Reply:
Yes, maybe you should rethink this one.